modules/virt-creating-localnet-nad-cli.adoc

```diff
@@ -12,10 +12,33 @@ You can create a network attachment definition (NAD) which describes how to atta
 * You have access to the cluster as a user with `cluster-admin` privileges.
 * You have installed the OpenShift CLI (`oc`).
 * You have installed the Kubernetes NMState Operator.
-* You have created a `NodeNetworkConfigurationPolicy` object to map the OVN-Kubernetes secondary network to an Open vSwitch (OVS) bridge.
 
 .Procedure
 
+. Create a `NodeNetworkConfigurationPolicy` object to map the OVN-Kubernetes secondary network to an Open vSwitch (OVS) bridge:
++
+[source,yaml]
+----
+apiVersion: nmstate.io/v1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: mapping # <1>
+spec:
+  nodeSelector:
+    node-role.kubernetes.io/worker: '' # <2>
+  desiredState:
+    ovn:
+      bridge-mappings:
+      - localnet: localnet-network # <3>
+        bridge: br-ex # <4>
+        state: present # <5>
+----
+<1> The name of the configuration object.
+<2> Specifies the nodes to which the node network configuration policy is to be applied.
+<3> The name of the additional network from which traffic is forwarded to the OVS bridge. This attribute must match the value of the `spec.config.name` field of the `NetworkAttachmentDefinition` object that defines the OVN-Kubernetes additional network.
+<4> The name of the OVS bridge on the node. This value is required only if you specify `state: present`.
```

Comment:
hm, if I don't specify the state, it'll default to present - which requires the OVS bridge name.

Reply:
Changed to - This value is required if the `state` attribute is `present`.

Reply:
Perfect, it is more accurate. 
Thank you.